src/main/scala/cromwell/engine/package.scala

```diff
@@ -34,4 +34,9 @@ package object engine {
     override def toString: String = "Succeeded"
     override val isTerminal = true
   }
+
+  trait WorkflowDescriptor {
+    def name: String
+    def id: WorkflowId
+  }
```

Comment:
Also maybe I should mention that in my branch I'm only using the id portion (for log messages)